src/store/reducers/members.ts

```diff
@@ -31,28 +34,45 @@ const GENESIS_TRUST_OPS = [
 ];
 export const GENESIS_MEMBER = Symbol("GENESIS");
 
-function getDefaultMemberFields() {
+function getDefaultMemberFields(): OptionalMemberFields {
   return {
     balance: new Big(0),
     totalDonated: new Big(0),
     totalMinted: new Big(0),
     trustedBy: Set<MemberId>(),
     invited: Set<MemberId>(),
-    trusts: Set<MemberId>()
+    invitedBy: undefined,
```

Comment:
If you want to keep the lack of repetition, you can express this as  `undefined as MemberId | typeof GENESIS_MEMBER | undefined` and keep the old code

Reply:
TBH I like the explicit typedef. I find the above a little obtuse to parse lol